src/Region/NetworkSites.php

```diff
@@ -256,6 +275,35 @@ protected function getSiteValue(string $key)
                 }
                 break;
 
+            case 'terms':
+                $value = [];
+                $site_taxonomies = get_taxonomies();
+
+                foreach ( $site_taxonomies as $site_taxonomy ) {
```

Comment:
Code style issue: the whitespacing doesn't match the rest of the file's code formatting (even though we usually use the space before closing parentheses and space after opening parentheses). Also line 285.

Reply:
You are right. Will sort